airbyte-server/src/test/java/io/airbyte/server/apis/MicronautHealthCheck.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.apis;
+
+import io.airbyte.api.model.generated.HealthCheckRead;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.context.env.Environment;
+import io.micronaut.core.util.StringUtils;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+@MicronautTest
+@Requires(property = "mockito.test.enabled",
+          defaultValue = StringUtils.TRUE,
+          value = StringUtils.TRUE)
+@Requires(env = {Environment.TEST})
+@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+class MicronautHealthCheck extends BaseControllerTest {
+
+  @Test
+  void testHealth() throws IOException {
+    Mockito.when(healthCheckHandler.health())
```

Comment:
The lack of this mocking is the reason why we were runnign in a NotFound error. It was not because the route was not register but because the response was null which lead micronaut to return a notFound error.